targets/unity-v2/templates/PlayFabSettings.cs.ejs

```diff
@@ -48,6 +48,14 @@ namespace PlayFab
         }
 #endif
 
+        private static string _uniqueId;
+        public static string DeviceUniqueIdentifier
+        {
+            get { return _uniqueId; }
+            set { _uniqueId = value; }
+        }
```

Comment:
This shouldn't be a property, and the underlying field shouldn't exist.
It should just be:
public static String DeviceUniqueIdentifier;